index.html

```diff
@@ -27,16 +27,25 @@ <h2 class="cta">Find events near you</h2>
       </div>
     </div>
   </div>
-  <div class="map">
-    <h2>Imagine a map</h2>
-  </div>
+  <div class="map" id="map"></div>
 
   <!-- FIXME bundle third party JS dependencies and serve them ourselves. 
        For now, just put them in here.  -->
   <script src="https://api.mapbox.com/mapbox.js/v3.0.1/mapbox.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/moment.min.js"></script>
   <!-- etc -->
 
-  <script src="./js/index.js"></script>
+  <!-- First draw map -->
+  <script src="./js/map-init.js"></script>
+
+  <!-- Then grab events data -->
+  <script src= "https://s3.amazonaws.com/thirdbear-backups/aclu/public/action_events.js"></script>
+
+  <!-- And valid zip code data -->
+  <script src="./data/us_postal_codes.js"></script>
```

Comment:
I transformed the zipcode data out of CSV and into a giant JS object because it was inconvenient to deal with as CSV. Included both the old and new files in this PR for reference, along with the script to do the transformation in case it needs to happen again. This should probably be served from s3 or something.